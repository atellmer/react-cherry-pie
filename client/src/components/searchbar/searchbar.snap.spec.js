import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmSearchbar from './index';


test('Jest: TmSearchbar (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();
  const tree = shallowRenderer.render(<TmSearchbar/>);

  expect(tree).toMatchSnapshot();
});
