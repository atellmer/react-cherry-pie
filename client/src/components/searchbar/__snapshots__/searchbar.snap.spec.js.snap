// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest: TmSearchbar (Snapshot) 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <ActionSearch
      color="#fff"
    />
  </div>
  <div
    className={undefined}
  >
    <input
      className={undefined}
      onBlur={[Function]}
      onFocus={[Function]}
      placeholder="Поиск"
      type="text"
    />
  </div>
</div>
`;
